README.md

```diff
@@ -43,10 +43,11 @@ You can tell logspout to ignore specific containers by setting an environment va
 
 Using the [httpstream module](http://github.com/gliderlabs/logspout/blob/master/httpstream), you can connect with curl to see your local aggregated logs in realtime. You can do this without setting up a route URI.
 
-	$ docker run -d -p 8000:8000 \
-		-v=/var/run/docker.sock:/var/run/docker.sock \
-		progrium/logspout
-	$ curl $(docker port `docker ps -lq` 8000)/logs
```

Comment:
I don't understand "the dynamic ip lookup". 

This line makes `--expose` on the CLI unnecessary:
https://github.com/gliderlabs/logspout/blob/master/Dockerfile#L4

Unless you have access to the Docker bridge, you need to use `--publish`.